ui/admin/app/styles/app.scss

```diff
@@ -299,6 +299,10 @@
     align-items: flex-end;
     display: flex;
     gap: sizing.rems(m);
+
+    [class*='hds-form-field--layout'] {
```

Comment:
What is the purpose of overriding HDS styles here?

Reply:
@calcaide the `Hds` wrapper for text input field has some predefined margin on the bottom that is causing the button and the input field to not line up correctly, this selector removes that margin.
<img width="2270" alt="Screenshot 2024-02-27 at 09 23 29" src="https://github.com/hashicorp/boundary-ui/assets/24277002/d62e697c-8314-43df-bf41-d14502c3e42b">